config/locales/en/providers.yml

```diff
@@ -1055,6 +1057,14 @@ en:
           we_love_manchester_emergency_fund: We Love Manchester Emergency Fund
           london_emergencies_trust: The London Emergencies Trust
           none_of_these: My client has received none of these payments
+      add_other_state_benefits:
+        show:
+          page_title: Does your client receive any other benefits?
+          caption: Client's means assessment
```

Comment:
Minor: In my branch I have added this to the generic content file as it appears on so many pages, but when these changes are merged I can just update this in my branch